examples/nlp/nmt_tutorial.py

```diff
@@ -104,7 +104,7 @@
     # source and target use different tokenizers, set tie_weight to False
     tie_weight = False
 else:
-    nf.logger.info(
+    nemo.logging.info(
```

Comment:
warning

Reply:
Changed this to raise ValueError instead since it does a exit(1) after